lua/codeium/source.lua

```diff
@@ -182,11 +182,14 @@ function Source:complete(params, callback)
 
 	self.server.request_completion(
 		{
+			text = text,
 			editor_language = filetype,
 			language = language,
-			cursor_offset = cursor_offset,
-			text = text,
+			curson_position = { row = cursor.row, col = cursor.col },
```

Comment:
Should this be `cursor_position`?

Reply:
ah yes, thank you. I should have caught that. If you can, you can make a PR to fix it. Otherwise I can get to it soon.